src/select.tsx

```diff
@@ -294,7 +295,8 @@ const Select: React.FC<SelectProps> = ({
             }
         },
     };
-    const {isDesktopOrBigger} = useScreenSize();
+
+    const iconSize = pxToRem(20);
```

Comment:
Icon size was smaller than in other input fields. Double check if this makes sense @yceballost

Reply:
it is expected